src/Microsoft.ML.DnnImageFeaturizer.AlexNet/AlexNet.cs

```diff
@@ -0,0 +1,50 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using System.IO;
+using System.Reflection;
+
+namespace Microsoft.ML.Transforms
+{
+    /// <summary>
+    /// This is an extension method to be used with the <see cref="DnnImageFeaturizerEstimator"/> in order to use a pretrained AlexNet model.
+    /// The NuGet containing this extension is also guaranteed to include the binary model file. Note that when building the project
+    /// containing this extension method, the corresponding binary model will be downloaded from the CDN at
+    /// https://aka.ms/mlnet-resources/image/AlexNetPrepOnnx/AlexNetPreprocess.onnx and
+    /// https://aka.ms/mlnet-resources/image/AlexNetOnnx/AlexNet.onnx  and placed into the local app directory
+    /// folder under mlnet-resources.
```

Comment:
does this comment about `Note that when building the project containing this extension method` still apply?

This doesn't make sense for an XML summary comment on a class. The XML comments are for the users of this API, not developers working on this file.

Reply:
Fixed the comment, thanks!

---
In reply to: [234754997](https://github.com/dotnet/machinelearning/pull/1447#discussion_r234754997) [](ancestors = 234754997)